### PR TITLE
Added routes and data of videos

### PR DESCRIPTION
**Pull request by nishasen**
Added routes for different pages in Video library
- Landing page
- explore
- like
- watch later
- playlist
- history
- single video page
- single playlist page

### CODE REVIEW

File: vibes/src/App.js [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
+        <Route path="/mockman" element={<MockAPI/>} />
+        <Route path="/login" element={<Login />} />
+        <Route path="/signup" element={<Signup/>} />
+        <Route path="/explore" element={<Explore />} />
+        <Route path="/explore/video/:id" element={<SingleVideo/>} />
+        <Route path="/history" element={<RequireAuth><History/></RequireAuth>} />
+        <Route path="/liked" element={<RequireAuth><Liked /></RequireAuth>} />
+        <Route path="/playlist" element={<RequireAuth><Playlist/></RequireAuth>} />
+        <Route path="/playlist/:id" element={<RequireAuth><PlaylistListing/></RequireAuth>} />
+        <Route path="/watchlater" element={<RequireAuth><WatchLater/></RequireAuth>} />
```
Comment (kjais1720) [inline vibes/src/App.js]: You can directly wrap the protected routes withing the RequireAuth component and then use `<Outlet/>` inside it to render the correct component

```suggestion
<Route element={<RequireAuth />}>
                <Route path="/liked" element={<Liked />} />
                <Route path="/history" element={<History />} />
                <Route path="/playlists" element={<Playlists />} />
                <Route path="/playlists/:id" element={<PlaylistListing />} />
                <Route path="/watchlater" element={<WatchLater />} />
              </Route>
```

File: vibes/src/App.js [hunk 2]
```diff
@@ -1,17 +1,30 @@
...
+      </Routes>
     </div>
```
Comment (therajatg) [inline vibes/src/App.js:28]: you could have all your routes in  a different file

**Review by therajatg — APPROVED** [review]
everything else looks good

**Review by kjais1720 — APPROVED** [review]
Everything looks good, just a small suggestion made.

File: vibes/src/App.js (hunk 2 above)
Comment (nishasen) [reply vibes/src/App.js:28]: Yes will change it while adding P1 features, will also add footers

**Pull request merged**